COPYRIGHT

```diff
@@ -1,4 +1,4 @@
-© Copyright 2015-2021 Adobe. All rights reserved.
+© Copyright 2015-2022 Adobe. All rights reserved.
```

Comment:
Per Adobe Open Source discussion, let's make this "Copyright 2022 Adobe. All rights reserved." (remove copyright symbol and year range, leaving only current year).